contrib/gitian-descriptors/gitian-win-signer.yml

```diff
@@ -6,15 +6,15 @@ suites:
 architectures:
 - "amd64"
 packages:
-# Once osslsigncode supports openssl 1.1, we can change this back to libssl-dev
```

Comment:
I actually think we should prefer being specific here, so that we know at a glance what version we're using.
I assume there's a ```libssl1.1-dev``` as well?

Reply:
I agree that more specific would be preferred, however from what I can see, it doesn't look like it. 
In a `ubuntu:bionic` container:
```bash
apt search libssl
libssl-dev/bionic-updates,bionic-security,now 1.1.1-1ubuntu2.1~18.04.4 amd64 [installed]
libssl1.0-dev/bionic-updates,bionic-security 1.0.2n-1ubuntu5.3 amd64
libssl1.0.0/bionic-updates,bionic-security,now 1.0.2n-1ubuntu5.3 amd64 [residual-config]
libssl1.1/bionic-updates,bionic-security,now 1.1.1-1ubuntu2.1~18.04.4 amd64
```

So no `libssl1.1-dev` package, only `lib1.0-dev` or `libssl-dev` (which is at 1.1.1).

Reply:
this makes sense, debian tends to only have versioned `-dev` packages when supporting an old version